src/main/scala/mesosphere/marathon/core/launcher/impl/InstanceOpFactoryImpl.scala

```diff
@@ -92,7 +92,7 @@ class InstanceOpFactoryImpl(
             hostPorts = ports.flatten
           )
 
-          taskOperationFactory.launchEphemeral(taskInfo, task)
+          taskOperationFactory.launchEphemeral(taskInfo, task, Instance(task)) // TODO PODS replace Instance(task)
```

Comment:
Nit: you could as well build the instance using its default ctor above and get rid of using the custom ctor right now. Not necessary for this PR though.